lib/ruby_lsp/server.rb

```diff
@@ -0,0 +1,157 @@
+# typed: strict
+# frozen_string_literal: true
+
+require_relative "internal"
+
+module RubyLsp
```

Comment:
We should be able to remove this module.

Reply:
If we remove the module, most of the constants below will need to have `RubyLsp` namespace, which I think doesn't help the readability.